openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -467,6 +461,22 @@ def test_set_credit_requirement_status(self, mode):
         course_requirements = api.get_credit_requirements(self.course_key)
         self.assertEqual(len(course_requirements), 2)
 
+        return course_requirements
+
+    @ddt.data(
+        *CourseMode.CREDIT_ELIGIBLE_MODES
+    )
+    def test_set_credit_requirement_status(self, mode):
+        username = self.user.username
+        credit_course = self.add_credit_course()
+
+        # Enroll user and verify his enrollment.
+        self.enroll(self.user, self.course_key, mode)
+        self.assertTrue(CourseEnrollment.is_enrolled(self.user, self.course_key))
+        self.assertTrue(CourseEnrollment.enrollment_mode_for_user(self.user, self.course_key), (mode, True))
```

Comment:
Here mode(VERIFIED, PROFESSIONAL, NO_ID_PROFESSIONAL_MODE) and active status of enrollment is checked. If you think we don't need to check that, and only checking if 'user is enrolled in course' is enough, please let me know.
Thanks.